libraries/standard/mqtt/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Include filepaths for source and include.
+include(filePaths.cmake)
+
+# MQTT library target.
+add_library( mqtt
+             ${MQTT_SOURCES}
+             include/mqtt.h
+             include/mqtt_lightweight.h )
+
+# MQTT public include path.
+target_include_directories( mqtt PUBLIC ${MQTT_INCLUDE_PUBLIC_DIRS} )
+
+# Organization of MQTT in folders.
```

Comment:
(Minor) Please update the comment as "Organization of MQTT in IDE projects" as these functions are only used for IDE project generation